Source/WebKit/WebProcess/Plugins/PDF/WKAccessibilityPDFDocumentObject.h

```diff
@@ -41,19 +43,20 @@ class WeakPtrImplWithEventTargetData;
 @interface WKAccessibilityPDFDocumentObject: NSObject {
     RetainPtr<PDFDocument> _pdfDocument;
     WeakObjCPtr<NSObject> _parent;
-    ThreadSafeWeakPtr<WebKit::PDFPluginBase> _pdfPlugin;
+    ThreadSafeWeakPtr<WebKit::UnifiedPDFPlugin> _pdfPlugin;
 }
 
 @property (assign) WeakPtr<WebCore::HTMLPlugInElement, WebCore::WeakPtrImplWithEventTargetData> pluginElement;
 
 - (id)initWithPDFDocument:(RetainPtr<PDFDocument>)document andElement:(WebCore::HTMLPlugInElement*)element;
 - (void)setParent:(NSObject *)parent;
 - (void)setPDFDocument:(RetainPtr<PDFDocument>)document;
-- (void)setPDFPlugin:(WebKit::PDFPluginBase*)plugin;
+- (void)setPDFPlugin:(WebKit::UnifiedPDFPlugin*)plugin;
 - (PDFDocument *)document;
-- (NSObject *)parent;
+- (NSObject *)accessibilityParent;
 - (id)accessibilityHitTest:(NSPoint)point;
 - (void)gotoDestination:(PDFDestination *)destination;
+- (NSRect)convertFromPDFPageToScreenForAccessibility:(NSRect)rectInPageCoordinate pageIndex:(WebKit::PDFDocumentLayout::PageIndex)pageIndex;
```

Comment:
I don't think this is the selector you want to implement? Is anyone calling into this?

I see that PDFKit expects `convertFromPDFViewToScreenForAccessibility`

Reply:
Yeah the PDFKit part of change is in https://stashweb.sd.apple.com/projects/IMG/repos/pdfkit/pull-requests/1137/overview

Reply:
Very sorry for the confusion, I assumed the necessary changes had landed.